lib/ocpp/Kconfig

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2023 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+config OCPP
+	bool "Open Charge Point Protocol"
+	depends on JSONC
+	help
```

Comment:
If this depends on websocket, you should add dependency here

Reply:
We have networking libraries in `subsys/net/lib` directory, should this one placed there?